### DISCUSSION TITLE
GitHub Actions job marked as SUCCESS but CYPRESS tests fail

### DISCUSSION THREAD

**Discussion by Monica89898** [cypress-io/cypress#28784] [CI setup]
My GitHub Actions run has always been marked a Success even when my cypress spec file test blocks marked as failed.
Below is the screenshot of how my yml file roughly looks like
![IMG_2908](https://github.com/cypress-io/cypress/assets/157529989/968e8595-0e45-48dd-80f3-a6fcde01885e)

**Comment by MikeMcC399** [comment]
- This is a duplicate to the issue posted to https://github.com/cypress-io/github-action/issues/1116

@Monica89898 

Please try to avoid posting the same question in two different places in the Cypress environment.
